config/webhook/manifests.yaml

```diff
@@ -13,7 +13,7 @@ webhooks:
       namespace: system
       path: /validate-infrastructure-cluster-x-k8s-io-v1alpha3-awsmachinetemplate
   failurePolicy: Fail
-  name: vawsmachinetemplate.kb.io
+  name: validating.awsmachine.infrastructure.x-k8s.io
```

Comment:
validating or validate or validation?

Reply:
This file is generated

Reply:
I don't really have a preference either way, happy to go with either of those.